source/services/facebook.js

```diff
@@ -3,8 +3,8 @@
  */
 
 export default {
-    counterUrl: 'https://graph.facebook.com/?id={url}&callback=?',
-    convertNumber: (data) => data.share.share_count,
+    counterUrl: 'https://graph.facebook.com/?id={url}&fields=og_object%7Bengagement%7D&callback=?',
```

Comment:
Nice, but I would restrict the query only to the data which actually used:
`'https://graph.facebook.com/?id={url}&fields=og_object%7Bengagement%7Bcount%7D%7D&callback=?'`

Reply:
Thank you for the suggestion! I have updated the code.